tensorflow/core/util/mkl_util.h

```diff
@@ -666,57 +666,63 @@ inline void ExecutePrimitive(const std::vector<primitive>& net,
 }
 
 template <typename T>
-inline Tensor ConvertMklToTF(OpKernelContext* context, const Tensor& mkl_tensor,
-                             const MklDnnShape& mkl_shape) {
-  Tensor output_tensor;
+inline Status ConvertMklToTF(OpKernelContext* context,
+                             const Tensor& input_mkl_tensor,
+                             const MklDnnShape& input_mkl_shape,
+                             Tensor& output_tf_tensor) {
```

Comment:
According to our [C++ style guide](https://google.github.io/styleguide/cppguide.html#Reference_Arguments), the output argument should be a pointer, i.e., `Tensor* output_tf_tensor` instead of `Tensor& output_tf_tensor`. Sorry for the inconvenience!

Reply:
@penpornk The one in `mkl_tfconv_op.h` is using `ConvertMklToTf()` and not `ConvertMklToTF()` (the former is defined locally). Same for `mkl_quantized_ops_test.cc` and `mkl_input_conversion_op.cc`.
I've updated the calls in `mkl_concat_op.cc` and `mkl_lrn_op.cc` since they are using the one I just modified.

Reply:
> According to our [C++ style guide](https://google.github.io/styleguide/cppguide.html#Reference_Arguments), the output argument should be a pointer, i.e., `Tensor* output_tf_tensor` instead of `Tensor& output_tf_tensor`. Sorry for the inconvenience!

No worries! I'll change it to be a pointer.

Reply:
@penpornk I have changed the output argument of `ConvertMklToTF()` to be a pointer. Please review. Thanks!

Reply:
I think there is a misunderstanding. The output argument of `ConvertMklToTf()` should be a pointer and the function signature should be
```c++
inline Status ConvertMklToTF(OpKernelContext* context,
                             const Tensor& input_mkl_tensor,
                             const MklDnnShape& input_mkl_shape,
                             Tensor* output_tf_tensor) {
  ...
  *output_tf_tensor = ...;
}
```
instead of
```c++
inline Status ConvertMklToTF(OpKernelContext* context,
                             const Tensor& input_mkl_tensor,
                             const MklDnnShape& input_mkl_shape,
                             Tensor& output_tf_tensor) {
  ...
  output_tf_tensor = ...;
}
```


And `ConvertMklToTf` should be called like this
```c++
Tensor output;
ConvertMklToTf<T>(context, input_mkl_tensor, input_mkl_shape, &output);
```
instead of 
```c++
Tensor output;
ConvertMklToTf<T>(context, input_mkl_tensor, input_mkl_shape, output);
```